test/Microsoft.DotNet.ShellShimMaker.Tests/ShellShimMakerTests.cs

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+using FluentAssertions;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.DotNet.ShellShimMaker.Tests
+{
+    public class ShellShimMakerTests : TestBase
+    {
+        [Fact]
+        public void GivenAnExecutablePathItCanGenerateShimFile()
```

Comment:
Should it say `ItCanGenerateAndExecuteShimFile`?

Reply:
actually shell is executing the shim file. I think executing it with shell (therefor verify shim works) should be part of the assertion.